packages/server/src/scim.ts

```diff
@@ -83,6 +88,11 @@ interface ScimGroupPatch {
     Operations: ScimGroupPatchOperation[];
 }
 
+interface ScimOrg {
+    users: ScimUser[];
+    groups: ScimGroup[];
+}
```

Comment:
This is the "virtual" scim model of the org. We'll use this to store and resolve ids and track changes.